plugin/lsp.vim

```diff
@@ -26,6 +26,10 @@ def g:LspServerReady(): bool
   return lsp.ServerReady()
 enddef
 
+def g:LspServerStatus(ftype: string): bool
```

Comment:
What is the use case for this new function?  Can you update the help text with a description
of this function and where this can be used?